nifi-commons/nifi-expression-language/src/main/java/org/apache/nifi/attribute/expression/language/evaluation/functions/JsonPathEvaluator.java

```diff
@@ -46,7 +47,14 @@ public JsonPathEvaluator(final Evaluator<String> subject, final Evaluator<String
         Object result = null;
         try {
             result = documentContext.read(compiledJsonPath);
+        } catch (PathNotFoundException pnf) {
+            // it is valid for a path not to be found, keys may not be there
+            // do not spam the error log for this, instead we can log debug
+            LOGGER.debug("PathNotFoundException for JsonPath " + compiledJsonPath.getPath(), pnf);
```

Comment:
Should be wrapped in a `isDebuggedEnabled` check

Reply:
Thanks!  Done